cli/server/server.go

```diff
@@ -491,7 +491,7 @@ func startServer(ctx *cli.Context) error {
 
 	go serv.Start(errChan)
 	if !cfg.ApplicationConfiguration.RPC.StartWhenSynchronized {
-		rpcServer.Start()
+		go rpcServer.Start()
```

Comment:
We have the same bottleneck few lines below when new RPC server is created on SIGHUP.

Reply:
done